src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy

```diff
@@ -207,12 +217,13 @@ class GatlingPluginExtension {
         @Input
         @Optional
         boolean getBatchMode() {
-            if (batchMode) {
-                return batchMode
-            } else {
-                def systemBatchMode = System.getProperty(BATCH_MODE_PROPERTY)
-                return systemBatchMode ? Boolean.parseBoolean(systemBatchMode) : false
-            }
+            return batchMode || Boolean.parseBoolean(System.getProperty(BATCH_MODE_PROPERTY))
```

Comment:
Boolean.getBoolean(BATCH_MODE_PROPERTY)

@Isammoc @guilgaly noobs 😂

Reply:
lol, I even already used that in the SBT plugin but not this one 😅